src/main/java/fi/aalto/cs/apluscourses/intellij/model/SettingsImporter.java

```diff
@@ -73,14 +73,14 @@ public void importIdeSettings(@NotNull Course course) throws IOException {
    * options to import, this function does nothing.
    */
   @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
-  public void importVMOptions(@NotNull Course course) {
+  public void importVMOptions(@NotNull Course course) throws IOException {
     if (!VMOptions.canWriteOptions()) {
       return;
     }
 
     var options = course.getVMOptions();
     for (var option : options.entrySet()) {
-      VMOptions.writeOption(option.getKey(), "=", option.getValue());
+      VMOptions.setProperty(option.getKey(), option.getValue());
```

Comment:
I don't see such method, am I looking in the wrong place?

Reply:
Do you have the latest version of the platform code?